wp-content/themes/inf_theme/bin/test.js

```diff
@@ -0,0 +1,235 @@
+const fs = require('fs');
+const path = require('path');
+const replace = require('replace-in-file');
+const emoji = require('node-emoji');
+const ora = require('ora');
+const chalk = require('chalk');
+const prompt = require('prompt-sync')();
+const {exec} = require('promisify-child-process');
+
+const output = require('./setup/output');
+
+const rootDir = path.join(__dirname, '..');
+
+const label = (msg) => console.log(chalk.cyan(msg));
+const error = (msg) => console.log(`${chalk.bgRed('Error')}${chalk.red(' - ')}${chalk.red(msg)}`);
+const log = (msg) => console.log(msg);
+const logWithPadding = (msg) => console.log(`    ${msg}`);
+
+const findReplace = async(findString, replaceString) => {
+  const regex = new RegExp(findString, 'g');
+  const options = {
+    files: `${rootDir}/**/*`,
+    from: regex,
+    to: replaceString,
+    ignore: [
+      path.join(`${rootDir}/node_modules/**/*`),
+      path.join(`${rootDir}/.git/**/*`),
+      path.join(`${rootDir}/.github/**/*`),
+      path.join(`${rootDir}/vendor/**/*`),
+      path.join(`${rootDir}/bin/rename.js`),
+      path.join(`${rootDir}/bin/rename-runnable.js`),
+      path.join(`${rootDir}/bin/setup.js`),
+      path.join(`${rootDir}/bin/setup-wp.js`),
+      path.join(`${rootDir}/bin/output.js`),
+      path.join(`${rootDir}/bin/files.js`),
+    ],
+  };
+
+  if (findString !== replaceString) {
+    await replace(options);
+  }
+};
+
+/**
+ * Prompts a user for something. 
+ *
+ * @param {object} settings 
+ */
+const promptFor = (settings) => {
+  let userInput;
+  label(settings.label);
+  do {
+    userInput = prompt(settings.prompt);
+
+    if (userInput.length <= 0) {
+      error(settings.error);
+    }
+  }
+  while (userInput.length <= 0 && userInput !== 'exit');
+  label('');
+  if (userInput === 'exit') {
+    log('Exiting script...');
+    process.exit();
+  }
+
+  return userInput;
+};
+
+const promptThemeData = () => {
+  let confirmed = 'n';
+  let newManifest = {};
+  
+  // -----------------------------
+  //  Prompt for project info
+  // -----------------------------
+
+  do {
+    /*
```

Comment:
How come this is commented out? It's not needed or?

Reply:
@dingo-d It's a WIP (and I'm pushing because I'll have to test soon) so better if you ignore it for a while. I'll let you know when it's ready.